server/broadcast.go

```diff
@@ -8,6 +8,7 @@ import (
 	"net/url"
 	"strings"
 	"sync"
+	"time"
```

Comment:
For future reference, the changes in this file could be put into its own commit. Not mandatory to be fixed up for this PR, but it's generally good Git practice to keep unrelated changes separate. (I'm OK with bundling commits that are only semi-related in the same PR.)